pkg/fileutil/files.go

```diff
@@ -146,3 +147,19 @@ func Contains(elems []string, param string) (exist bool, position int) {
 	}
 	return false, 0
 }
+
+// This function finds all files of a specific extension
+func GetFilesWithSpecificExtension(folderPath, ext string) []string {
+	var files []string
+	filepath.Walk(folderPath, func(path string, f os.FileInfo, _ error) error { //nolint:errcheck
+		if f != nil && !f.IsDir() {
+			r, err := regexp.MatchString(ext, f.Name())
```

Comment:
Questioning why keep the error here as we are not doing anything useful when we get an error. If it is not important that we got an error while matching extensions, would you consider below instead:

```
r, _ := regexp.MatchString(ext, f.Name())
```

Reply:
If it errors out, then it would skip that file check and move on. Need that to be able to identify files with specific extension